### DISCUSSION TITLE
[자료구조 1일차] 배열, 연결리스트

### DISCUSSION THREAD

**Discussion by seongeun42** [Shortudy-10th/tech-for-developer#30] [오늘의 주제]
배열, 연결리스트 관련 질문 2인조 1개, 3인조 2개씩 총 5개

**Comment by watty-exclamationMark** [comment]
배열과 연결리스트는 데이터 모음을 다루는 자료구조로, 논리적 연속성을 가진다는 공통점이 있습니다.
하지만 물리적(메모리) 연속성을 따지게 되면 둘은 차이점이 생기는데 이를 설명해주세요.

**Comment by watty-exclamationMark** [comment]
Static Array와 Dynamic Array의 차이점은 무엇일까요?

**Comment by watty-exclamationMark** [comment]
Array는 Random Access Data Structure의 하나의 예가 될 수 있다고 합니다. 
Random Access란 무엇일까요?

**Comment by watty-exclamationMark** [comment]
연결리스트의 종류에 대해 설명해주세요.

**Comment by sikutisa** [comment]
배열과 연결리스트의 특성에 기반하여, 각 자료구조를 사용하는 것이 다른 자료구조를 사용하는 것보다 유리한 상황에 대해 설명해주세요.

**Comment by LireEruel** [comment]
연결리스트에서 요소를 추가하거나 삭제하는 과정을 설명해주세요

**Comment by Jihn0118** [comment]
배열과 연결리스트의 특징에 대해서 설명해주세요
